src/libraries/react-calendar/components/miniature/CalendarSmall.styled.ts

```diff
@@ -0,0 +1,108 @@
+import { styled } from '@mui/system'
+import Grid from '@mui/material/Grid'
+import Typography, { TypographyProps } from '@mui/material/Typography'
+import { lightBlue } from '@mui/material/colors'
+
+export const Section = styled('section')({
+  minHeight: 264,
+  minWidth: 240,
+  background: 'white',
+})
+
+export const Title = styled(Typography)({
+  marginLeft: 8,
+  textTransform: 'capitalize',
+})
+
+export const Navigation = styled(Grid)({
+  marginRight: 4,
+})
+
+export const DayHeader = styled(Typography)({
+  textAlign: 'center',
+  fontSize: 12,
+  color: lightBlue[800],
+  lineHeight: '26px',
+  padding: 2,
+  borderColor: 'white',
+  borderStyle: 'solid',
+  textTransform: 'capitalize',
+  background: 'white',
+  height: 34.3,
+  width: 34.3,
+})
+
+interface StyledDayProps extends TypographyProps {
+  selected: boolean
+  today: boolean
+  isCurrentMonth: boolean
+}
+
+type ElementColors = {
+  text: string
+  background: string
+}
+
+const colors = {
```

Comment:
Technically all this colors that you are using in the application shoudl be inside your theme, for example we have `theme.pallete.common.white` if you need white color and usually you have your primary paleete so in this case `lightBlue[700]` would be `theme.pallete.primary.main` and so on. For now you can leave this as it is, but in the future it would be good to move this to the theme configuration.